<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>195</width>
    <height>119</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lunar Club Tools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="MoonInfoTab" name="moonInfoTab">
       <attribute name="title">
        <string>Moon Info</string>
       </attribute>
      </widget>
      <widget class="FeaturesTab" name="featuresTab">
       <attribute name="title">
        <string>Features</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>195</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>Configure</string>
    </property>
    <addaction name="actionLocation"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionLunarClubTools"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuConfigure"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/exit.svg</normaloff>:/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit the program</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionLocation">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/location.svg</normaloff>:/location.svg</iconset>
   </property>
   <property name="text">
    <string>Location</string>
   </property>
   <property name="toolTip">
    <string>Configure observation location</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionLunarClubTools">
   <property name="text">
    <string>Lunar Club Tools</string>
   </property>
   <property name="toolTip">
    <string>About Lunar Club Tools</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MoonInfoTab</class>
   <extends>QWidget</extends>
   <header>widgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FeaturesTab</class>
   <extends>QWidget</extends>
   <header>widgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
